Soccar [Client]/src/nl/soccar/library/SessionController.java

```diff
@@ -102,13 +105,17 @@ public void leave(Session session, Player player) {
         return Collections.unmodifiableList(allSessions);
     }
 
+    /**
+     * Method that gets a observable list of all rooms.
+     * 
+     * @return Observablelist of all rooms.
+     */
     public ObservableList<Room> getAllRooms() {
         ObservableList<Room> roomList = FXCollections.observableArrayList();
-        for (Session s : allSessions) {
+        allSessions.stream().forEach((s) -> {

```

Comment:
.stream() is overbodig. Ook kan '(s)' veranderd worden naar 's'. Curly brackets zijn ook niet nodig. Nieuwe code: forEach(s -> roomList.add(s.getRoom());